primitives/src/assets.rs

```diff
@@ -474,6 +474,6 @@ where
 			)
 			.map(|_| ())
 		}
-		.map_err(|e| FungibleLedgerError::InvalidTransfer(e))
```

Comment:
Why not propagate the dispatch error ? It will have useful info.

Reply:
See previous conversation between @Garandor  and @Dengjianping